examples/soundspeed/atlas/ex_atlases_auto_query.py

```diff
@@ -10,26 +10,76 @@
 from hyo2.soundspeedmanager.qt_callbacks import QtCallbacks
 
 
+class ModelOptions(IntEnum):
+    # World Models
+    WOA09 = 1
+    WOA13 = 2
+    RTOFS = 3
+
+    # Regional Models
+    # East Coast
+    CBOFS = 10  # RG = True     # Format is GoMOFS
+    DBOFS = 11  # RG = True     # Format is GoMOFS
+    GoMOFS = 12  # RG = True     # Format is GoMOFS
+    NYOFS = 13  # RG = False
+    SJROFS = 14  # RG = False
+
+    # Gulf of Mexico
+    NGOFS = 20  # RG = True     # Format is GoMOFS
+    TBOFS = 21  # RG = True     # Format is GoMOFS
+
+    # Great Lakes
+    LEOFS = 30  # RG = True     # Format is GoMOFS
+    LHOFS = 31  # RG = False
+    LMOFS = 32  # RG = False
+    LOOFS = 33  # RG = False
+    LSOFS = 34  # RG = False
+
+    # Pacific Coast
+    CREOFS = 40  # RG = True     # Format is GoMOFS
+    SFBOFS = 41  # RG = True     # Format is GoMOFS
+
+
+# Setup Logger
 logging.basicConfig(level=logging.DEBUG,
                     format="%(levelname)-9s %(name)s.%(funcName)s:%(lineno)d > %(message)s")
 logger = logging.getLogger(__name__)
 
-switch = "LEOFS"  # WOA09 or WOA13 or RTOFS or GoMOFS or LEOFS or LHOFS
+# Choose Model
+switch = 21  # Choose a ModelOptions Value to test
```

Comment:
The hard-coding of an integer is not the intent of having an enumeration. Use `ModelOptions.TBOFS`!